manim/mobject/types/vectorized_mobject.py

```diff
@@ -879,6 +880,49 @@ def get_subcurve(self, a, b):
         vmob.pointwise_become_partial(self, a, b)
         return vmob
 
+    def get_direction(self):
+        """
+        Uses shoelace in space_ops.py to calculate the direction.
+        The direction of points determines in which direction the
+        object is drawn, clockwise or counterclockwise.
+
```

Comment:
I think this is an excellent method for adding a doctest, something like this:

```suggestion

        Examples
        --------
        The default direction of a :class:`~.Circle` is counterclockwise::
        
        >>> from manim import Circle
        >>> Circle().get_direction()
        "CCW"
```

(I *think* CCW is correct here, but please correct it if I am wrong; I can't try it out right now.)

Reply:
CCW is correct, that's the default direction. Committing.